Automattic/jetpack
# Instant Search: Improve Accessibility of Instant Search options form.

Moves the separator "•" from CSS into the DOM and sets the aria-hidden attribute so that the separators are ignored by screen readers.

Fixes #20791 

Adds the `aria-current="true"` value to the currently selected search option making it accessible to screen readers.

Fixes #23366.

#### Changes proposed in this Pull Request:


- Removes the CSS that added the "•" separators.
- Adds them inline to the HTML.
- Updated CSS to maintain styling.
- Adds `aria-hidden` attribute to hide from screen readers.
- Adds the `aria-current` attribute to the current option.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Load the Instant Search overlay.
2. Inspect the "•" separators.
3. Confirm they all have `aria-hidden="true"` set.
4. Test in Firefox using the Accessibility inspector.
5. Confirm the separators are not visible there.

![NoSeps](https://user-images.githubusercontent.com/40267301/180377673-1ce58c1d-327c-4bf6-9978-378494c97141.png)

1. Select one of Relevance, Newest, Oldest.
2. Open the inspector and confirm that the `aria-current` value is set to `true`.
3. Change sort options and confirm the attributes update accordingly.

![AriaCurrentSnap](https://user-images.githubusercontent.com/40267301/180382615-879c43e5-4c18-492c-9afa-6b87e3b383bb.png)
